storage/FileManagerPosix.cpp

```diff
@@ -86,6 +88,7 @@ size_t FileManagerPosix::numSlots(const block_id block) const {
   }
 
   if ((file_stat.st_size % kSlotSizeBytes) != 0) {
+    LOG(ERROR) << "Size of file " << filename.c_str() << " was not integral.";
```

Comment:
This log is not related to this bug. Please remove it.

As an alternative, please remove the below exception, and use `LOG(FATAL)` instead.